changes/issue-3079-remove-host-counts

```diff
@@ -0,0 +1 @@
+* Remove host counts on software temporarily to provide faster results
```

Comment:
How temporary is this? Just thinking about how the changelog would read, is that something that will come back in a different implementation? We might want to clarify that in the message.